stylus/components/forms.styl

```diff
@@ -357,8 +350,8 @@ $radio
             content '•'
             color white
             text-align center
-            font-size .7rem
-            line-height .7rem
+            font-size 1rem
+            line-height .9rem
```

Comment:
Why is the line-height different from the font-size ? The line-height does not round to pixels cleanly (0.9*16 = 14.4px) , I suggest to change it to 0.9375rem or 0.875rem.

Reply:
Well, browsers round up but you're right, it'll be cleaner.

Reply:
> Well, browsers round up

All browsers do not have the same strategy for rounding up or down, and it seems some css properties have subpixel rendering.

Source: https://stackoverflow.com/questions/34676263/sub-pixels-calculated-and-rendered-differently-among-browsers and any "subpixel rendering" search in Google